SuperBuild/External_Autoscoper.cmake

```
@@ -93,6 +93,16 @@ if(NOT DEFINED ${proj}_DIR AND NOT ${SUPERBUILD_TOPLEVEL_PROJECT}_USE_SYSTEM_${p
 
   set(${proj}_DIR ${EP_BINARY_DIR}/Autoscoper-build)
 
+  # Launcher Settings
+  set(${proj}_LIBRARY_PATHS_LAUNCHER_BUILD ${EP_BINARY_DIR}/Autoscoper-build/bin/$<CONFIG>) # Autoscoper executable
+  list(APPEND ${proj}_LIBRARY_PATHS_LAUNCHER_BUILD ${EP_BINARY_DIR}/GLEW-Install/bin/) # Glew library
+  list(APPEND ${proj}_LIBRARY_PATHS_LAUNCHER_BUILD ${EP_BINARY_DIR}/TIFF-Install/bin/) # TIFF library
+
+  mark_as_superbuild(
+    VARS ${proj}_LIBRARY_PATHS_LAUNCHER_BUILD
+    LABELS "LIBRARY_PATHS_LAUNCHER_BUILD"
+    )
+
```

```suggestion
  #-----------------------------------------------------------------------------
  # Launcher setting specific to build tree

  # library paths
  set(${proj}_LIBRARY_PATHS_LAUNCHER_BUILD
    ${EP_BINARY_DIR}/Autoscoper-build/bin/$<CONFIG> # Autoscoper executable
    ${EP_BINARY_DIR}/GLEW-Install/${Slicer_THIRDPARTY_BIN_DIR} # Glew library
    ${EP_BINARY_DIR}/TIFF-Install/${Slicer_THIRDPARTY_BIN_DIR} # TIFF library
    )

  #-----------------------------------------------------------------------------
  # Launcher setting specific to install tree
  
  # NA
```

This should allows the settings to be correct across platform and avoid hard-coding directory name.